src/locales/en/modals/translation.json

```diff
@@ -41,6 +41,13 @@
 		"line2": "Warning: you will not be able to recover it.",
 		"title": "Delete the account?"
 	},
+	"deleteOrgUser": {
+		"cancelButton": "Cancel",
+		"deleteButton": "Delete",
+		"line1": "The action is irreversible.",
+		"line2": "Warning: you will not be able to recover it.",
+		"title": "Delete the user?"
```

Comment:
Make it same format as all other delete modals (for example triggers):


**Delete User**

Are you sure you want to delete [username]?

This action cannot be undone, and all related data will be permanently removed.__

Reply:
changed